pkg/controller/util.go

```diff
@@ -82,6 +84,19 @@ func getSecretName(client kubernetes.Interface, pvc *v1.PersistentVolumeClaim) (
 	return name, nil
 }
 
+// returns the PVC size with fs overhead percentage included.
+func getPvcSize(pvc *v1.PersistentVolumeClaim) (string, error) {
```

Comment:
Will this actually be used anywhere else besides the import controller?  It might be good to keep it local in that file rather than continuing to dump random things in this util.go file.